addons/binding/org.openhab.binding.keba/src/main/java/org/openhab/binding/keba/handler/KeContactHandler.java

```diff
@@ -56,68 +60,69 @@
 import com.google.gson.JsonParser;
 
 /**
- * The {@link KeContactP20Handler} is responsible for handling commands, which
+ * The {@link KeContactHandler} is responsible for handling commands, which
  * are sent to one of the channels.
  *
  * @author Karel Goderis - Initial contribution
  */
-public class KeContactP20Handler extends BaseThingHandler {
+public class KeContactHandler extends BaseThingHandler {
 
     public static final String IP_ADDRESS = "ipAddress";
     public static final String POLLING_REFRESH_INTERVAL = "refreshInterval";
-
-    public static final int CONNECTION_REFRESH_INTERVAL = 100;
+    public static final int LISTENING_INTERVAL = 100;
     public static final int REPORT_INTERVAL = 2000;
     public static final int PING_TIME_OUT = 3000;
     public static final int BUFFER_SIZE = 1024;
     public static final int REMOTE_PORT_NUMBER = 7090;
-    public static final int LISTENER_PORT_NUMBER = 7090;
 
-    private Logger logger = LoggerFactory.getLogger(KeContactP20Handler.class);
+    final Logger logger = LoggerFactory.getLogger(KeContactHandler.class);
```

Comment:
Any reason to make the logger non-private, please keep them private and give each every class their own logger.